packages/okta-angular/README.md

```diff
@@ -101,13 +101,40 @@ Finally, import the `OktaAuthModule` into your `NgModule`, and instantiate it by
 export class MyAppModule { }
 ```
 
-### Reference
-**Configuration Options**
-  - `issuer` **(required)**: The OpenID Connect `issuer`
-  - `clientId` **(required)**: The OpenID Connect `client_id`
-  - `redirectUri` **(required)**: Where the callback is hosted
-  - `scope` *(optional)*: Reserved or custom claims to be returned in the tokens
-  - `onAuthRequired` *(optional)*: Accepts a callback to make a decision when authentication is required. If not supplied, `okta-angular` will redirect directly to Okta for authentication.
+## Use the Access Token
+When your users are authenticated, your Vue application has an access token that was issued by your Okta Authorization server. You can use this token to authenticate requests for resources on your server or API. As a hypothetical example, let's say you have an API that provides messages for a user. You could create a `MessageList` component that gets the access token and uses it to make an authenticated request to your server.
+
+Here is what the Angular component could look like for this hypothetical example:
+
+```typescript
+// messagelist.component.ts
+
+import { Component } from '@angular/core';
+import { Http, Headers, RequestOptions } from '@angular/http';
+import { OktaAuthService } from '@okta/okta-angular';
+import 'rxjs/Rx';
+
+@Component({
+  template: `
+    <div *ngIf="messages.length">
+      <li *ngFor="let message of messages">{{message.message}}</li>
+    </div>
+  `
+})
+export class MessageListComponent {
+  messages = [];
+  constructor(private oktaAuth: OktaAuthService, private http: Http) {
+    const headers = new Headers({ Authorization: 'Bearer ' + oktaAuth.getAccessToken().accessToken });
+    // Make request
+    this.http.get(
+      'http://localhost:{serverPort}/api/messages',
+      new RequestOptions({ headers: headers })
+    )
+    .map(res => res.json())
+    .subscribe((messages: Array<Object>) => messages.forEach(message => this.messages.push(message)));
+  }
+}
+```
```

Comment:
You could also use an HttpInterceptor with Angular 4.3+ and HttpClient.

Reply:
Great suggestion! I want to keep this consistent with our [quickstart](https://developer.okta.com/quickstart/#/angular/nodejs/generic) guide for now, but if the community is using `HttpClient` over the `Http` module, we should update both of our docs.

Just out of curiosity, is `HttpClient` the favorable choice?

Reply:
Yes, `Http` is deprecated in Angular 5.